.github/workflows/CIMaster.yml

```diff
@@ -0,0 +1,92 @@
+name: CI
```

Comment:
So we basically need to have separate CI files just for the schedule and slack notifications? :(

Reply:
https://github.community/t/reusing-sharing-inheriting-steps-between-jobs-declarations/16851

Looks like this has been a thing for a while... annoying.